clientconn.go

```diff
@@ -840,6 +840,72 @@ func (ac *addrConn) connect() error {
 	ac.cc.handleSubConnStateChange(ac.acbw, ac.state)
 	ac.mu.Unlock()
 
+	common := func() bool {
```

Comment:
Not sure if there's a better name for this?

Reply:
How about `ac.reconnect`?  Note that this doesn't access any locals, so it doesn't need to be a lambda.

Reply:
Done